QUEST.tsv

```diff
@@ -1184,22 +1184,22 @@ QUEST_20150317_001183	Having now lost the gold bullion given to him by the stran
-QUEST_20150317_001192	Model Stars, who obtained to say that it was a wagon with injuries to bet, was to ensure that not a very very words that are collectively injury as words that originally had in the car. {nl} try and words that were injured look with it without split and many injuries, became good feeling is that he saw the big obtained. {nl}
-QUEST_20150317_001193	Having obtained the word one two Phil and wagon mind, thinking that there is not a bad situation at the moment, this time because it is the turn of the quality, it was decided to unless really careful. {nl} So, we decided to go to the place people are less in place plans that tried to visit soldiers. It has been moved for a while and led the horse and wagon to hold the reins at such a thought {nl}. However, there is no guarantee that do not meet in person as it where there is no popular. {nl}
-QUEST_20150317_001194	Also model Stars Mr. uncommon was hit with many of the Alchemist of age in the unpopular mountain path. {nl} old Alchemist, you have a big bullion on hand was a roadside stone chair three sitting. {nl} the old Alchemist when we model Stars Mr. passing has entered the hand rather than bullion, I thought that lead. {nl}
-QUEST_20150317_001195	It is thought to "this Alchemist is really? The research to change the lead to gold" such a thing, and tried to hit through him. {nl} is Alchemist is, had been a long time called whether model Stars' people was glad that pass in front of their own. And, we have proposed a suddenly betting game. Somehow today there was a face that get bored even much model Stars Mr. It is suggested that you do to try all the world model Stars san and bet. {nl} but model Stars san or like betting, and to find a bet of opponent I to was also not small so easily. {nl}
-QUEST_20150317_001196	Really so not like model Stars', it has been also considered whether to reject this thing, that thing that took a cheap was moving the model Stars' mind. {nl} old Alchemist is because he walked the lead in hand. {nl}
-QUEST_20150317_001197	This time, the other person in a different location Once you because unconditional quality of rented one of the lead, was to feel that one is not a decent selection compared more a large amount of money to the possibility of Sonsuru. {nl} model Stars Mr. Make sure that you are even bring later because there is no money immediately if the quality, the results corresponding to the thing, was Mashi of course. {nl}
+QUEST_20150317_001187	Someone approached Modestas, who had been lying down hurt, and helped him get back on his feet. He turned out to be one of the people standing on the side, the Wagon Owner. {nl}The Wagon Owner who helped Modestas uttered with relief.{nl}"I am glad you are not too hurt. And your horse also didn't hurt himself tripping over that pit."{nl}Modestas noticed a well hidden pit where the Wagon Owner pointed.{nl}The Wagon Owner continued.{nl}
```

Comment:
Don't think the commas are necessary in the first sentence:
"Someone approached Modestas who had been lying down hurt and helped him get back on his feet."

"The Wagon Owner who helped Modestas uttered with relief," (comma instead of period before quoting speech text)